xskillscore/core/deterministic.py

```diff
@@ -413,7 +436,7 @@ def effective_sample_size(a, b, dim='time', skipna=False, keep_attrs=False):
     .. [1] Bretherton, Christopher S., et al. "The effective number of spatial degrees of
       freedom of a time-varying field." Journal of climate 12.7 (1999): 1990-2009.
 
-    Wilks, Daniel S. Statistical methods in the atmospheric sciences. Vol. 100.
+    .. [2] Wilks, Daniel S. Statistical methods in the atmospheric sciences. Vol. 100.
```

Comment:
You could try taking the `.. [2]` out. As far as I understand if you number the references sphinx expects you to refer to the reference in the text e.g. [1]_. Given that we don't refer to it in the text it's probably best to leave it as an unnumbered reference.

Reply:
Ah good catch. I forgot that I had done an inline citation in the notes. I'll just revert them both to a bullet list so we don't have to deal with this rst issue.